prompt2model/dataset_retriever/description_dataset_retriever.py

```diff
@@ -553,9 +731,6 @@ def retrieve_dataset_dict(
             or None if there are no relevant datasets.
         """
         sorted_list = self.retrieve_top_datasets(prompt_spec)
-        logger.info(f"Top datasets retrieved. Top datasets: {sorted_list}")
-        top_dataset_info = self.rerank_datasets(sorted_list, prompt_spec)
-        logger.info(f"Rerank completed. Top dataset info: {top_dataset_info}")
-        return self.canonicalize_dataset_automatically(
-            top_dataset_info, prompt_spec, auto_transform_data, num_points_to_transform
-        )
+        logger.info("Top datasets retrieved.")
```

Comment:
I think we can show the top datasets, but this is a minor nit

Reply:
I feel its a bit overwhelming amount of information - more useful for a developer than a user